static/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -31,6 +31,7 @@ import ProjectLink from './projectLink';
 
 type Props = {
   projects: Array<ReleaseProject>;
+  adoptionStages: {} | undefined;
```

Comment:
```suggestion
  adoptionStages?: Record<string, any>;
```
should we also move this prop to the bottom of this object, since it is optional?